src/AppBundle/Entity/Client.php

```diff
@@ -288,13 +303,13 @@ public function getFullname()
 
     public function setCourtDateWithoutTime($courtDate = null)
     {
-        $this->courtDate = ($courtDate instanceof \DateTime) ?
-                new \DateTime($courtDate->format('Y-m-d')) : null;
+        $this->courtDate = ($courtDate instanceof DateTime) ?
+                new DateTime($courtDate->format('Y-m-d')) : null;
```

Comment:
Is it possible to call format on the existing object? rather than creating new instance of DateTime?